src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/high-risk-data-breaches/[type]/HighRiskBreachLayout.test.tsx

```diff
@@ -6,41 +6,80 @@ import { it, expect } from "@jest/globals";
 import { getAllByRole, render, screen } from "@testing-library/react";
 import { composeStory } from "@storybook/react";
 import { axe } from "jest-axe";
+import { setupJestCanvasMock } from "jest-canvas-mock";
 
 import Meta, {
   BankAccountStory,
   CreditCardStory,
   PinStory,
   SsnStory,
+  HighRiskBreachDonePasswordsNextStory,
+  HighRiskBreachDoneSecurityQuestionsNextStory,
+  HighRiskBreachDoneSecurityTipsNextStory,
+  HighRiskBreachDoneNoNextStepStory,
 } from "./HighRiskDataBreach.stories";
 
+beforeEach(() => {
+  setupJestCanvasMock();
```

Comment:
Does this need to be called before every test, or can we just call it at the start of the test file (like regular `jest.mock`s)? If it needs to be called before every test (reading the docs, it looks like it might reset the mocks?), is this something that might be moved to `jest.setup.ts`, which [supports `beforeEach`](https://jestjs.io/docs/configuration#setupfilesafterenv-array), as far as I can see?

Reply:
It does need to be run before each test, because we are [resetting mocks](https://github.com/mozilla/blurts-server/blob/resolution-flow-celebration/jest.config.cjs#L145) in `jest.config.cjs` by default. We can also move the `beforeEach` to `jest.setup.ts`, but my intention is to only have `setupJestCanvasMock` in the test file it’s needed in.